PeculiarPedestrians_Feet.cs

```diff
@@ -86,6 +56,8 @@ public PeculiarPedestrians_Feet()
 			DisplayName = "Hooks for Feet ({{r|D}})";
 		}
 
+		public override bool UseVariantName => false; // Does not currently respect defect status
```

Comment:
Honestly, I've been thinking about dropping defect status for the mutation. It's as much of a defect as carapace is, though I suppose it's substantially less useful so maybe it's fine to keep.

Reply:
Very true but at least carapace can be upgraded, haha.